screens/WelcomePage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+
+export default function HomePage({ navigation }) {
+  return (
+    <View className="flex-1 items-center justify-center bg-gray-100">
+      <LinearGradient
+        colors={['#D8D9CE', '#FFFFFF']}
+        className="absolute inset-x-0 top-[26%] w-full h-[80%] rounded-t-[40px]"
+      />
+
+      <Image
+        source={require('../assets/kebab_logo.png')}
+        className="w-40 h-40 mb-5 z-10"
+        resizeMode="contain"
+      />
+
+      <Text className="text-2xl font-bold text-center mb-4 z-10 text-gray-800 uppercase">
+        Legnica Kebab City Tour
+      </Text>
+
+      <Text className="text-center text-sm text-gray-600 px-5 mb-8 z-10">
+        Legnica Kebab City Tour jest to aplikacja oraz witryna internetowa służąca pomocą w odnalezieniu lokalizacji wszystkich dostępnych, w planach oraz zamkniętych punktów gastronomicznych serwujących słynne Kebaby. Poniższe przyciski pokierują Cię dalej. Wybierz przycisk "Mapa" aby bezpośrednio odnaleźć Kebaby rozsiane po Legnicy. Możesz też utworzyć konto lub zalogować się aby dodać Twojego ulubionego Kebaba do zakładki "Ulubione", dzięki czemu łatwiej go odnajdziesz!!
+      </Text>
+
+      <TouchableOpacity
+        className="bg-[#283618] py-3 rounded-full mb-4 w-[75%] items-center"
```

Comment:
Done

Reply:
Not like this. You set up whatever you need inside a config file, it is global, you don't need to import this.
Then use it by calling the new name: bg-button-green

The only exception I would make, is with a gradient, it was good the old way
`colors={['#D8D9CE', '#FFFFFF']}`

Reply:
I changed it partially to the previous version, but with the buttons in react native it's the same situation as with the gradient. We need to use something like this: `   <Button
          title="Press me"
          color="#f194ff"
          onPress={() => Alert.alert('Button with adjusted color pressed')}
        />` It's example from docs, when I'm trying to add the bg-button-green instead of the color="#283618" to our button it doesn't work.

Reply:
I think it's done well now